public/pages/DefineDetector/components/DetectorInfo/DetectorInfo.tsx

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import { EuiFieldText, EuiTextArea } from '@elastic/eui';
+import { Field, FieldProps } from 'formik';
+import React from 'react';
+import ContentPanel from '../../../../components/ContentPanel/ContentPanel';
+import { getError, isInvalid } from '../../../../utils/utils';
+import { validateDetectorDesc } from './utils/validation';
+import { FormattedFormRow } from '../../../../components/FormattedFormRow/FormattedFormRow';
+
+interface DetectorInfoProps {
+  onValidateDetectorName: (detectorName: string) => Promise<any>;
+}
+function DetectorInfo(props: DetectorInfoProps) {
```

Comment:
Sure. I was keeping the same component name that's currently used in `createDetector` but now would be a good time to change it - agreed that it is vague. How about `DetectorNameAndDescription` or `NameAndDescription` or `NameDescription`? This could make it consistent with the `Timestamp` and `Datasource` components which are more descriptive.

Reply:
I will change to `NameAndDescription`